modules/network-observability-enriched-flows-kafka.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: Procedure
+[id="network-observability-enriched-flows-kafka_{context}"]
+= Export enriched network flow data
+
+You can send network flows to Kafka, so that they can be consumed by any processor or storage that supports Kafka input, such as Splunk, Elastic, or Fluentd,
+
+.Prerequisites
+* Installed Kafka
+* The `FlowCollector` `spec.agent` is set to `Kafka`.
```

Comment:
I missed that, this is wrong, there's no such requirement on `spec.agent`.
Just having installed Kafka. And then, configuring the `exporters` as explained below